### DISCUSSION TITLE
--config-path support for autofix settings

### DISCUSSION THREAD

**Discussion by seanulacra** [rome/tools#4780] [Suggestions]
Hi!

Our project is using Rome ( loving it so far by the way🙏 ) but we've been running into a weird case with autofix on save.

Recently many of us have been running into workflow issues with the noUnusedVariables rule while working locally. This is something we need to enforce at a higher level but while using the VSCode extension configuration

```
  "editor.codeActionsOnSave": {
    "quickfix.rome": true,
  },
  ```

There is no option to configure it further. We have a dozen other invocations of Rome commands that benefit from the default config file location - so it's counterintuitive for us to pass a --config-path to each of them just so the quick fix extension on VSCode can behave differently than our pre-push scripts for example which need to be more strict.

A simple change would be to allow us to declare the location of a configuration file for the quickfix command to use - so we could isolate specific requirements.

**Comment by Conaclos** [comment]
Hi!

Thanks to bring it to light!
The on-save action should not apply unsafe quick-fixes like the unsafe fix of `noUnusedVariables`. I will fix it for the up-coming release :)

This should solve your problem.

**Reply by Conaclos** [reply]
Biome is now [publicly announced](https://biomejs.dev/blog/annoucing-biome). You can test the fix on the [first stable version of Biome](https://biomejs.dev/blog/biome-v1).
